src/goto-symex/renaming_level.cpp

```diff
@@ -56,6 +56,28 @@ operator()(ssa_exprt ssa_expr, const namespacet &ns, unsigned thread_nr) const
   return renamedt<ssa_exprt, L0>{ssa_expr};
 }
 
+void symex_level1t::insert(ssa_exprt ssa, unsigned index)
+{
+  const irep_idt identifier = ssa.get_identifier();
```

Comment:
`const &` ?

Reply:
the problem is `ssa` is moved below, and the move will happen before we use the identifier, so we need a copy